docs/components/app.jsx

```diff
@@ -29,9 +29,9 @@ class App extends React.Component {
   render() {
     return (
       <div style={{display: "flex", minHeight: "100vh", flexDirection: "column"}}>
-        <Header
-          text={"Interested in using Victory on your next project? Let’s talk."}
-        />
+        <Header backgroundColor="#ebe3db">
```

Comment:
Please use `VictorySettings.palestSand` instead of the literal value. That goes for all `backgroundColor` props below as well.